archimedes/archimedes/runner.py

```diff
@@ -245,23 +267,35 @@ def start(self, request: RunRequest, exec_script_path: str) -> LocalProcess:
 
         stderr_log_io = open(stderr_log, 'wb')
         return subprocess.Popen(
-            [shutil.which('python'), exec_script_path],
+            [self._cmd(request), exec_script_path],
             stderr=stderr_log_io,
             env={k: v for k, v in [s.split('=', maxsplit=1) for s in environment]},
         ), stderr_log_io, stderr_log
 
+    def _cmd(self, request: RunRequest):
+        if request.run_with("python"):
+            return shutil.which('python')
+        else:
+            # If we want to support alternate language  engines for local development,
+            #   we'll have to pull the binary and archimedes-node
+            #   into the archimedes image via a multi-stage docker build.
+            # Otherwise, you just have to test via Vulcan.
+            raise ValueError(
```

Comment:
For testing non-python scripts, I think you have to test it in the `archimedes-node` container itself, instead of via `archimedes-run` and the local isolator ... not sure if there's a way to make it work via the local isolator, but seems difficult without injecting Node, etc., all into `archimedes`, which seems like it could get messy.